_includes/cookies/cookie-consent.html

```diff
@@ -0,0 +1,63 @@
+<noscript>
+  <div id="cookie-notice-nojs">
+  <div class="container">
+  <div id="cookie-choice-nojs">
```

Comment:
```suggestion
  <div class="container">
  <div id="cookie-choice-nojs">
```

Can we delete the outer div?